.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,23 +6,41 @@
   data retrieval. 
 -->
 
-## Description
+## Related issues
 
-- Link to related issues: `#<issue number>`
-- Description of the problem the contribution solves or link to design issue
-  - [ ] Explanation of how existing TDS code falls short
-  - [ ] Explanation of backwards compatibility for users who are already on this feature
+See <!-- put issue number here, or delete this section -->
 
-## Considerations
+## Notes
 
-- [ ] Provide access to finalised designs in Sketch and Invision
-- [ ] Add documentation
+* Feature 1
+* Feature 2
+* Patch 1
+* Patch 2
 
-## Dev Acceptance Criteria
+## Merge Notes
```

Comment:
Is the audience for this template supposed to be the core team or all contributors? If it is all contributors I don't think this "merge notes" section makes sense. As an outside contributor it seems likely that I won't know or care whether my code should be rebased or squashed or why I should be specifying it.

Perhaps the merging notes stuff is something that maintainers should know about and gets documented elsewhere, and we keep that complexity hidden away from outside contributors.